tests/ldap/conftest.py

```diff
@@ -9,11 +9,38 @@
 
 from __future__ import absolute_import, print_function
 
+import os
+
+import jinja2
 import pytest
+from flask import Blueprint
 from invenio_app.factory import create_app as _create_app
 
 
 @pytest.fixture(scope="module")
 def create_app():
     """Create test app."""
     return _create_app
+
+
+@pytest.fixture(scope="module")
+def app_with_mail(app):
+    """App with email test templates."""
+    # app.register_blueprint(
+    #     Blueprint(
+    #         "invenio_app_ils_tests",
+    #         __name__,
+    #         template_folder="../cds_books/circulation/templates",
+    #     )
+    # )
+    # add extra test templates to the search app blueprint, to fake the
+    # existence of `invenio-theme` base templates.
+    test_templates_path = os.path.join(os.path.dirname(__file__), "templates")
+    print("PATH:", test_templates_path)
+    test_templates_path = "/home/konstantina/Documents/Work/cds-books/cds_books/circulation/templates"
```

Comment:
This seems to be for local testing, or am I missing something?

Reply:
Yes you are right, I forgot it